### ISSUE TITLE
IntelliSense crashes in C file

### ISSUE THREAD

**Issue by lh123** [microsoft/vscode-cpptools#2913]
**Type: LanguageService**

- OS: Archlinux
- GCC: 8.2.1

## Step to reproduce
1. Create a 'C' file._(This crash only happens in the 'C' file)_
```c
int main(void) {
}
```

2. Declare a function before main.
```c
static void test(int);// IntelliSense crashes when I type 'int'

int main(void) {
}
```

3. C/C++ Output.
`Failed to read response from server: 8`

**Comment by sean-mcmanus** [comment]
Thanks for reporting this. The bug was already fixed in our pending 0.21.0-insiders2 update (see https://github.com/Microsoft/vscode-cpptools/issues/1785 ) which we plan to release any day now (as soon as we finish doc comments). This has been our top hitting crash according to our telemetry, but we never had a repro until now, so we're now able to verify that the fix we made actually works, and maybe later we can fix the root cause :)

**Comment by sean-mcmanus** [comment]
The crash when entering params in C declarations should be fixed now with our latest Insiders release: https://github.com/Microsoft/vscode-cpptools/releases . If you're using VS Code Insiders or have http.proxySupport not set to "off", you might not get the 0.21.0-insiders2 update automatically with C_Cpp.updateChannel set to "Insiders".
